packages/lib/services/plugins/RepositoryApi.ts

```diff
@@ -3,7 +3,7 @@ import shim from '../../shim';
 import { PluginManifest } from './utils/types';
 const md5 = require('md5');
 const compareVersions = require('compare-versions');
-
+const packageInfo = require('../../../app-desktop/packageInfo.js');
```

Comment:
I don't think there should be a dependency from lib to app-desktop

Reply:
Yes. That's why I closed this PR.
I will raise another PR after rectifying it.